src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -38,4 +38,70 @@
 	<bean id="customerLoadService" parent="customerLoadService-parentBean"
 		class="edu.cornell.kfs.module.ar.batch.service.impl.CuCustomerLoadServiceImpl" />
 
+	<bean id="contractsGrantsInvoiceCreateDocumentService" parent="contractsGrantsInvoiceCreateDocumentService-parentBean"/>
+	<bean id="contractsGrantsInvoiceCreateDocumentService-parentBean" class="edu.cornell.kfs.module.ar.batch.service.impl.CuContractsGrantsInvoiceCreateDocumentServiceImpl" abstract="true">
```

Comment:
Based on your other code changes, I don't think you need to override the parent bean. If all you need to change is the "class" attribute, then you can override it on the main "contractsGrantsInvoiceCreateDocumentService" bean so that you don't need to modify the parent bean.